src/utils.js

```diff
@@ -595,6 +618,12 @@ function resolveDependencies(schema, definitions, formData) {
         dependencyValue
       );
     }
+    return processDependencies(
```

Comment:
Makes this function recursive so that once a dependency is resolved, it goes through all the dependencies again.